pipeline-notebooks/pipeline-section.ipynb

```diff
@@ -1007,7 +1008,10 @@
     "            section_elements = sec_document.get_section_narrative(regex_enum)\n",
     "            results[f\"REGEX_{i}\"] = section_elements\n",
     "    if response_type == \"application/json\":\n",
-    "        return {section:convert_to_isd(section_narrative) for section, section_narrative in results.items()}\n",
+    "        if response_schema == \"labelstudio\":\n",
```

Comment:
`response_schema="isd", ` should be added to the signature of `pipeline_api` (before the `m_` params)

Reply:
Thanks for catching that, just added and re-comitted.

Reply:
Was there a reason why `response_schema` had to be added before the `m_` params? I had to rearrange the order to pass `CI / lint` and would like to know if the order matters.

Reply:
more of a convention that `response_schema` and `response_type` are more commonly used pipeline_api keyword args than `m_` ones. lint complaining about the formatting to make multiline makes sense. are you sure about it complaining about ordering?

Reply:
Because we added a `ValueError` for anything other than isd and labelstudio, exceptions were raised because there were some test cases where it was treated as if `response_schema=[RISK_FACTORS]`.